src/fvmcts/fv_mcts_vanilla.jl

```diff
@@ -315,7 +318,7 @@ function simulate(planner::FVMCTSPlanner, node::FVStateNode, depth::Int64)
     ucb_action = coordinate_action(mdp, planner.tree, s, planner.solver.exploration_constant, node.id)
 
     # Monte Carlo Transition
-    sp, r = gen(DDNOut(:sp, :r), mdp, s, ucb_action, rng)
+    sp, r = @gen(:sp, :r)(mdp, s, ucb_action, rng)
```

Comment:
Is this what you meant by the new POMDPs.jl API? No more DDN??

Reply:
Yes, as of POMDPs 0.9 we decided to get rid of the flexible DDN.